### ISSUE TITLE
Exporting bert model to a saved model format

### ISSUE THREAD

**Issue by pcnfernando** [google-research/bert#843]
I have been referring to [predicting_movie_reviews_with_bert_on_tf_hub.ipynb ](https://github.com/google-research/bert/blob/master/predicting_movie_reviews_with_bert_on_tf_hub.ipynb) to create a sentiment analysis on the IMDB dataset.

I need to export it to tensorflow saved model format so that I can integrate it with a separate system.
Can you please guide me to achieve it?

**Comment by Cumberbatch08** [comment]
yes, I also fine-tune a classification model use bert hub way. But I don't know how to reload the trained model? Did you solve that?

**Comment by emrecalisir** [comment]
Is your problem solved? I am having the same problem.

**Comment by Cumberbatch08** [comment]
> Is your problem solved? I am having the same problem.

Hi, after fine-tuning, just use the saved output file, don't need the hub module.

**Comment by AMZzee** [comment]
> > Is your problem solved? I am having the same problem.
> 
> Hi, after fine-tuning, just use the saved output file, don't need the hub module.

Hi, how does your output file look like? what type of file? and how did u export it? also please tell me how to load it into another program.

**Comment by jageshmaharjan** [comment]
Maybe take a look at this 
https://medium.com/delvify/bert-rest-inference-from-the-fine-tuned-model-499997b32851
